### ISSUE TITLE
Method detail

### ISSUE THREAD

**Issue by CikZ2023** [The-Martyr/CausalMM#5]
Hello, I have a question: Taking the attention intervention of LLM as an example, if we randomize the attention of LLM, although it blocks the influence of modality priors, it may lead to the generated content being incomprehensible because the attention weights themselves carry certain semantic information. Moreover, I tried to use contrastive decoding to subtract the logits after randomization from the normal logits, but the results were not good, and the generated content was gibberish. Could you please explain my confusion?

**Comment by The-Martyr** [comment]
This is a very interesting point in our work, and I also discussed this with many people during the poster stage of ICLR. I guess you used a few examples to do a case study on the contrastive decoding method. In fact, the causal inference properties of our work tend to be more statistically optimized, that is, overall improvement (such as benchmark evaluation), which is in line with its mathematical expectations. But it may perform poorly or even weaker than the baseline in some cases.

Hope this helps.

**Comment by CikZ2023** [comment]
Is your method about directly intervening in the attention of LLMs through counterfactual reasoning, such as setting the attention weights to random values or mean values, to mitigate hallucinations? I personally don't quite understand the effectiveness of this approach, because when I tested setting the attention weights to random values on LLaVA, the results were almost all gibberish. Could you please explain the principle behind this method to me?
